backend/user/views.py

```diff
@@ -95,14 +95,15 @@ def post(self, request):
         title = request.data.get("title", None)
         body = request.data.get("body", None)
         delay = request.data.get("delay", 0)
+        is_dev = request.data.get("isDev", False)
```

Comment:
nit: I don't know if this is already set in stone for iOS, because if so we can leave it. But maybe we should snake case `isDev` if there's not a huge hassle.